README_JP.md

```diff
@@ -0,0 +1,290 @@
+  - width, height: 置換後のテクスチャのサイズ。いずれも 2 のべき乗である必要があります。置換元のテクスチャのサイズと合わせる必要はありません。
+  
+- new GDIHelper(*width*, *height*);
+  テクスチャの描きかえには System.Drawing.Bitmap を使用できますが、Bitmap を直接描きかえる処理の実装は複雑で難易度が高いため、補助を行うための GDIHelper を提供しています。
+
+  GDIHelper のインスタンスはテクスチャ毎に生成してください。
+
+  - GDIHelper.Graphics
+
+    テクスチャに紐づけられた System.Drawing.Graphics を取得します。Graphics を利用すると円や文字列、アルファチャンネル（透過）付画像などを描画することができます。
+  
+    **Graphics.DrawImage メソッドは遅いため推奨しません。GDIHelper.DrawImage メソッドを使用してください。**
+  
+  - GDIHelper.DrawImage(*bitmapToDraw*, *x*, *y*)
+    GDIHelper.DrawImage(*bitmapToDraw*, *x*, *y*, *sourceOffsetY*, *height*)
+  
+    GDI+ による Graphics.DrawImage メソッドは遅いため、GDI32 を利用して Bitmap を描画する代替メソッドを提供しています。
+    ただし、このメソッドの実行前に BeginGDI() メソッド、実行後に EndGDI() メソッドを実行する必要があることに注意してください。
+  
+  - GDIHelper.FillRect12(*color*, *x1*, *y1*, *x2*, *y2*)
+    GDIHelper.FillRectWH(*color*, *x1*, *y1*, *width*, *height*)
+
+    単一色で塗り潰された長方形を描画します。
+    このメソッドの実行前に BeginGDI() メソッド、実行後に EndGDI() メソッドを実行する必要があることに注意してください。
+  
+  BeginGDI() メソッドを実行してから EndGDI メソッドを実行するまでの間に GDIHelper.Graphics を使用しないでください。
+  また、BeginGDI() メソッドを実行してから EndGDI メソッドを実行するまでの間以外で GDIHelper.DrawImage メソッド、GDIHelper.FillRect から始まる名前のメソッドを使用しないでください。
+  
+- TextureHandle.ShouldUpdate(*time*, *fps*)
+
+  `time` で指定した現在時刻を基に、`fps` で指定したフレームレート (テクスチャの更新頻度) の下限を下回っているかを取得します。
+  テクスチャの更新はコストが大きいため、できるだけ頻度を下げることが推奨されます。
+
+  このメソッドは単に「最後の更新からどのくらいの時間が経過したか」を確認しているのみなので、このメソッドを使わず、強制的に更新することもできます。
+
+- TextureHandle.Update(*gdiHelper or bitmap*)
+
+  ビデオカードに GDIHelper や Bitmap をプッシュし、TextureHandle に紐づけられたゲーム内のテクスチャを更新します。
+  テクスチャの更新はコストが大きいため、更新頻度の最適化のために TextureHandle.ShouldUpdate メソッドを使用することを推奨します。
+
+- TextureManager.Dispose()
+
+  プラグインのアンロードの際は、全リソースの解放を忘れないようにしてください。
+
+## タッチ
+
+コード例 (重要な部分のみであり、完全なコードではありません) :
+
+```csharp
+// using System;
+// using System.Windows.Forms;
+// using Zbx1425.DXDynamicTexture;
+
+private static TouchTextureHandle hTIMSTex;
+
+[DllExport(CallingConvention.StdCall)]
+public static void Load(){
+    TextureManager.Initialize();
+    hTIMSTex = TouchManager.Register("foo/bar/tims_placeholder.png", 512, 512);
+    hTIMSTex.SetClickableArea(0, 0, 400, 300);
+    TouchManager.EnableEvent(MouseButtons.Left, TouchManager.EventType.Down);
+    hTIMSTex.MouseDown += HTIMSTex_MouseDown;
+}
+
+private static void HTIMSTex_MouseDown(object sender, TouchEventArgs e) {
+    MessageBox.Show(String.Format("X: {0}, Y: {1}", e.X, e.Y));
+}
+```
+
+動的テクスチャのコードとほぼ同じ処理になります。相違点としては TouchManager.Register メソッドを使用することと、同メソッドの戻り値が TouchTextureHandle 型であることくらいです。
+TouchTextureHandle は TextureHandle を継承しているため、動的テクスチャのための機能は TouchTextureHandle にも含まれます。
+
+また、同一のファイルを TouchManager と TextureManager の両方に登録しないでください。TouchManager.Register メソッドには TextureManager.Register メソッドと同一の機能が含まれます。
+
+また、タッチされた位置を色で検出するため、CabIlluminance による運転台の明るさの設定には対応しないことをご了承ください。DaytimeImage と NighttimeImage で同一のファイルを指定してください。
```

Comment:
It is not to say that CabIlluminance cannot be used, but the displayed color of the texture must not change despite the changes in CabIlluminance. So the train developer should set the same value for `DaytimeImage` and `NighttimeImage`.